app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,27 @@
+class LikesController < ApplicationController
+  before_action :set_post
+
+  def new
+    @like = Like.new
+  end
+
+  def create
+    @like = Like.new
+    @like.post = @post
+    @like.user = current_user
+
+    if @like.save
+      flash[:notice] = 'Like was successfully created.'
+      redirect_to user_post_path(@like.post.author_id, @like.post.id)
+    else
+      flash.now[:error] = 'Oops, something went wrong'
+      render :new
+    end
+  end
```

Comment:
- [ ] Nice work implementing the create action for adding like to a post. :clap: However there is not way for the user to like a post. So I suggest that you add a button `like` in the `app/views/posts/show.html.erb` view, which will be responsible of add a like to a post. 
  **HIT:** You can use the `button_to` helper
```erb
<%= button_to 'likes', user_post_likes_path(@post, @user)%>
```